pkg/config/errors.go

```diff
@@ -43,6 +60,15 @@ func (err *InvalidSettingError) Error() string {
 	return fmt.Sprintf("invalid setting provided to a module, cause: %v", err.cause)
 }
 
+// InvalidModuleError signifies a problem with the supplied module name.
+type InvalidModuleError struct {
+	modID string
```

Comment:
Make it exported, `ModuleID string`, or even `ModuleID ModuleID`

Reply:
Done